main.rb

```diff
@@ -0,0 +1,36 @@
+# Importing classes
```

Comment:
use rubocop, please.
https://github.com/rubocop-hq/rubocop
your app must pass all check, only the ASCII symbols can stay in comments